sim/global-types.ts

```diff
@@ -73,6 +73,7 @@ type PokemonSet = {
  * - S = event, 3rd char+ is the index in .eventPokemon
  * - D = Dream World, only 5D is valid
  * - V = Virtual Console transfer, only 7V is valid
+ * - G = LGPE transfer, only 7G is valid
```

Comment:
I would use 8V for this. For the same reasons VC transfer uses 7V rather than 1V and 2V.

Reply:
I guess using 8G is fine. We're not _super_ low on letters. But if we are one day, we might want to consolidate them.

Reply:
Yeah it was a typo